extensions/html/server/tsconfig.json

```diff
@@ -4,8 +4,9 @@
 		"module": "commonjs",
 		"outDir": "./out",
 		"noUnusedLocals": true,
+		"sourceMap": true,
 		"lib": [
-			"es5", "es2015.promise"
+			"es5", "es2015.promise", "dom"
```

Comment:
Is that intended?

Reply:
When I ran `tsc -p server` without this I got millions of errors like this:
```
lib/jquery.d.ts(2769,29): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2792,16): error TS2304: Cannot find name 'HTMLElement'.
lib/jquery.d.ts(2804,34): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2817,37): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2825,39): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2838,38): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2845,25): error TS2304: Cannot find name 'HTMLElement'.
lib/jquery.d.ts(2850,12): error TS2304: Cannot find name 'HTMLElement'.
lib/jquery.d.ts(2861,35): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2873,22): error TS2304: Cannot find name 'HTMLElement'.
lib/jquery.d.ts(2881,37): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2887,22): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2920,41): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2932,22): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2940,39): error TS2304: Cannot find name 'Element'.
lib/jquery.d.ts(2971,43): error TS2304: Cannot find name 'Element'.
```

Reply:
You need to run `tsc -p server` in `/src`. That's where the `tsconfig.json` is.
Best is to always use `npm run compile` or 'npm run watch`

Reply:
If the `gulp compile-extension:html-server` is not using the `tsconfig` here then why do we have the `tsconfig` here?

Reply:
But if you don't like it we can change it back and put a dev README.md at `/extensions/html`, what do you think?

Reply:
tsconfig is still used by the typescript language server in VSCode.

I added an 'include' to tsconfig to lib and other folders are ignore, so you won't see the error anymore.

How to compile vscode (including html server) is described in [here](https://github.com/Microsoft/vscode/wiki/How-to-Contribute). `npm run watch` is what should be used.